docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -0,0 +1,130 @@
+[[configure-endpoint-integration-policy]]
+= Configure an integration policy
+
+After the {agent} is installed successfully with {endpoint-sec}, several protections features — including preventions against malware, ransomware, memory threats, and malicious behavior — are automatically enabled on protected hosts (a Platinum+ license feature). If needed, you can update the integration policy to configure protection settings, event collection, antivirus settings, and trusted applications to meet your organization's security needs.
+
+To configure the integration policy:
+
+1. In the {security-app}, go to **Manage** -> **Endpoints** to view the Endpoints list. Remember that you must have admin permissions in {kib} to access this page.
+2. From the **Policy** column, select the integration policy you want to configure. The integration policy configuration page appears.
+3. Review the following settings on the **Policy settings** tab and configure as appropriate:
+* <<malware-protection>>
+* <<ransomware-protection>>
+* <<memory-protection>>
+* <<behavior-protection>>
+* <<event-collection>>
+* <<register-as-antivirus>>
+* <<adv-policy-settings>>
+* <<save-policy>>
+
+4. Click the **Trusted applications** tab to review the list of <<trusted-apps-ov, trusted applications>> assigned to this integration policy. On this tab, you can:
+* Expand and view a trusted application — Click the arrow next to the application's name.
```

Comment:
I wanted to isolate each of these actions so the list would be easier to quickly scan, and I feel like converting them into full sentences slows down the scanability. But if the original em dashes don't work, I'm open to other options.

I'll "resolve" the other identical comments about these list items, just to clean up the PR review, but we can re-open them if needed.

Reply:
I figured you had structured these steps this way for that reason! And looking at them again, it might make more sense to prioritize scannability for these steps.